types.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-i18next';
```

Comment:
i'm not familiar enough with typescript to know about these declaration files, but I do know that the way create-react-app is supposed to handle typescript integration is to put a similar file into the `src` directory. See https://github.com/ledhed2222/gregweisbrod.com/blob/master/src/react-app-env.d.ts (from my personal site lol)